src/main/java/org/apache/tomcat/vault/util/PropertySourceVault.java

```diff
@@ -22,13 +22,15 @@
 
 package org.apache.tomcat.vault.util;
 
+import org.apache.catalina.Globals;
 import org.apache.juli.logging.Log;
 import org.apache.juli.logging.LogFactory;
 import org.apache.tomcat.util.IntrospectionUtils.PropertySource;
 import org.apache.tomcat.vault.security.vault.PicketBoxSecurityVault;
 import org.apache.tomcat.vault.security.vault.SecurityVault;
 import org.apache.tomcat.vault.security.vault.SecurityVaultException;
 import org.apache.tomcat.vault.security.vault.SecurityVaultFactory;
+import org.jasypt.util.text.BasicTextEncryptor;
```

Comment:
hm that is adding a dependency.... What is the licence of jasypt and url to know more.

Reply:
ASLv2.0 and http://jasypt.org/.